### PR TITLE
Ep/add effnetb1 272x240

### PR DESCRIPTION
**Pull request by pellegreene**

### CODE REVIEW

File: brainscore_vision/models/effnetb1_272x240/model.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                and 'P' not in pil_image.mode.upper():  # not binary and not alpha and not palletized
+            # work around to https://github.com/python-pillow/Pillow/issues/1144,
+            # see https://stackoverflow.com/a/30376272/2225200
+            return pil_image.copy()
+        else:  # make sure potential binary images are in RGB
+            rgb_image = Image.new("RGB", pil_image.size)
+            rgb_image.paste(pil_image)
+            return rgb_image
+
+def get_model_list():
```
Comment (mike-ferguson) [inline brainscore_vision/models/effnetb1_272x240/model.py:66]: can remove `get_model_list`

File: brainscore_vision/models/effnetb1_272x240/model.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        self.efnet_model = timm.create_model('tf_efficientnet_b1_ns', pretrained=True)
+
+    def forward(self, x):
+        x = self.efnet_model(x)
+        return x
+
+def get_model(name):
+    assert name == 'effnetb1_272x240'
+    model_tf_efficientnet_b1_ns= EffNetBX()
+    dir_path = os.path.dirname(os.path.realpath(__file__))
```
Comment (mike-ferguson) [inline brainscore_vision/models/effnetb1_272x240/model.py:82]: These weights should be hosted on S3

**Review by mike-ferguson — CHANGES_REQUESTED** [review]
Need to upload weights onto S3 and remove get_model_list

File: brainscore_vision/models/effnetb1_272x240/model.py (hunk 2 above)
Comment (pellegreene) [reply brainscore_vision/models/effnetb1_272x240/model.py:82]: It doesn't even look like this model is pulling in any new weights. Am I mistaken here?

**Comment by mike-ferguson** [comment]
god merged per me, Jenkins is running very slow (>12 hours)

**Pull request merged**